src/main/controllers/helpers/JudgmentHelpers.ts

```diff
@@ -0,0 +1,511 @@
+        },
+      });
+    }
+    responseDetails.push({
+      key: {
+        text: translations.copyCorrespondence,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecisionApplication?.value.copyToOtherPartyYesOrNo,
+      },
+    });
+  }
+  decisionDetails.push(
+    {
+      key: {
+        text: translations.decision,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecision?.value.enterNotificationTitle,
+      },
+    },
+    {
+      key: {
+        text: translations.dateSent,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecision?.value.date,
+      },
+    },
+    {
+      key: {
+        text: translations.sentBy,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecision?.value.decisionMadeBy,
+      },
+    }
+  );
+  if (selectedDecision?.value.additionalInformation) {
+    decisionDetails.push({
+      key: {
+        text: translations.additionalInfo,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecision?.value.additionalInformation,
+      },
+    });
+  }
+
+  if (selectedDecision?.value.responseRequiredDoc) {
+    decisionDetails.push({
+      key: {
+        text: translations.document,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        html: downloadLink,
+      },
+    });
+  }
+  decisionDetails.push(
+    {
+      key: {
+        text: translations.decisionMadeBy,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecision?.value.decisionMadeBy,
+      },
+    },
+
+    {
+      key: {
+        text: translations.name,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecision?.value.decisionMadeByFullName,
+      },
+    },
+    {
+      key: {
+        text: translations.sentTo,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: selectedDecision?.value.selectPartyNotify,
+      },
+    }
+  );
+  return [applicationDetails, responseDetails, decisionDetails];
+};
+
+export function getDecisions(userCase: CaseWithId): TseAdminDecisionItem[] {
+  const decisions = userCase?.genericTseApplicationCollection
+    .filter(
+      obj =>
+        !obj.value.type.includes(applicationTypes.respondent.c) &&
+        obj.value.copyToOtherPartyYesOrNo === YesOrNo.YES &&
+        obj.value.adminDecision?.length
+    )
+    .flatMap(obj => obj.value.adminDecision);
+  return decisions.filter(it => it.value.typeOfDecision === 'Judgment');
+}
+
+export const populateDecisionItemsWithRedirectLinksCaptionsAndStatusColors = (
+  decisionItems: TseAdminDecisionItem[],
+  url: string,
+  translations: AnyRecord
+): TseAdminDecisionItem[] => {
+  if (decisionItems && decisionItems.length) {
```

Comment:
same here, i won't post any more comments but just give it a scan yourself.

Reply:
Fixed